### ISSUE TITLE
QGIS 2.6 - zoom to extent wmts error

### ISSUE THREAD

**Issue by daliborjanak** [klokantech/tileserver-php#50]
It was reported that new version of qgis (2.6) can't open wmts from tileserver-php. We need to test it.

**Comment by hynekpridal** [comment]
after check and fix notify lorenc.cala@gmail.com

**Comment by klokan** [comment]
Possibly needs to be fixed also in WMTS gate.

**Comment by daliborjanak** [comment]
Qgis opens wmts correctly but have problem with bounding box of layer so "Zoom to layer" zooms to whole world. There will be problem maybe with crs transform (description in our wmts). I tryed combinate tags ```<WGS84BoundingBox>``` and ```<BoundingBox>```. But I haven't got good results. I need more tests.

It also doesn't work in QGIS 2.8. Old versions (2.x) don't have problem.

**Comment by daliborjanak** [comment]
Qgis changed behavior of zooming to WMTS layer. Qgis now zooms to the TileMatrixSet extent and not to extent of the layer. I found only this reports of this issue:
http://osgeo-org.1560.x6.nabble.com/Apparent-bug-or-undesirable-behaviour-in-QGIS-2-6-1-2-4-0-td5187337.html
http://isticktoit.net/?p=1021

I tried other WMTS servers (MapServer and Arcgis for Server) and they have same problem with zoom to layer. So this is not problem in tileserver-php.

@klokan, We can reported as bug or enhancement.
